pypowsybl/network.py

```diff
@@ -5329,37 +5366,37 @@ def get_unused_order_positions_after(network: Network, busbar_section_id: str) -
     return pd.Interval(left=positions[0], right=positions[1], closed='both')
 
 
-def replace_tee_point_by_voltage_level_on_line(network: Network, tee_point_line1: str, tee_point_line2: str,
-                                               tee_point_line_to_remove: str,
-                                               bbs_or_bus_id: str, new_line1_id: str, new_line2_id: str,
-                                               new_line1_name: str = None, new_line2_name: str = None) -> None:
+def replace_tee_point_by_voltage_level_on_line(network: Network,  df: _DataFrame = None, raise_exception: bool = False,
+                                               reporter: _Reporter = None, **kwargs: _ArrayLike) -> None:
```

Comment:
it will be a breaking change, won't it ?

Reply:
Yes it does, I made it so that all the functions would take a dataframe as input and not arguments straight. So it is a breaking change, which is why I am not sure if it is a good idea...

Reply:
The refactoring can be reused by creating the dataframes in the different functions but I felt like it made sense to have all the modifications taking dataframes as input but I am not sure.

Reply:
I agree and I think it is better this way but I'm not sure about the breaking change

Reply:
Is there something like deprecated in Python?

Reply:
there seems to be something https://pypi.org/project/Python-Deprecated/